src/presets/universal_deployer.cairo

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.9.0 (presets/universal_deployer.cairo)
+
+/// # UniversalDeployer Preset
```

Comment:
```suggestion
/// # UniversalDeployerContract Preset
```

Reply:
`UniversalDeployer Preset` (without the `Contract` suffix) is consistent with the other presets e.g. `ERC20 Preset`, no?

Reply:
The Standard ERC20 is not called ERC20Contract, but UDC has the C on the name, right? I don't like having contract fwiw, i rather just UniversalDeployer, but I think we should be consistent with the name.

Reply:
the standard/singleton is called Universal Deployer Contract, but on purpose the filename was UniversalDeployer.cairo. We can use both, let's call it UniversalDeployer preset
![image](https://github.com/OpenZeppelin/cairo-contracts/assets/1703831/259e78e3-00d8-410c-b11f-bd33ff036ee8)